docs/src/expose_pg_services.md

```diff
@@ -108,11 +108,11 @@ On Minikube you can setup the ingress controller running:
 minikube addons enable ingress
 ```
 
-Then, patch the `tcp-service` ConfigMap to redirect to the primary the
+Then, patch the `tcp-services` ConfigMap to redirect to the primary the
 connections on port 5432 of the Ingress:
 
 ```sh
-kubectl patch configmap tcp-services -n kube-system \
+kubectl patch configmap tcp-services -n ingress-nginx \
   --patch '{"data":{"5432":"default/cluster-example-rw:5432"}}'
 ```
```

Comment:
I think we don't need this section anymore since the `tcp-services` it's properly created in the `cluster-expose-service.yaml` file, can you confirm ?

Reply:
@djw-m what I'm saying it's that cluster-expose-service.yaml already fix this, look at this example:

```
$ minikube addons enable ingress
💡  ingress is an addon maintained by Kubernetes. For any concerns contact minikube on GitHub.
You can view the list of minikube maintainers at: https://github.com/kubernetes/minikube/blob/master/OWNERS
    ▪ Using image registry.k8s.io/ingress-nginx/controller:v1.7.0
    ▪ Using image registry.k8s.io/ingress-nginx/kube-webhook-certgen:v20230312-helm-chart-4.5.2-28-g66a760794
    ▪ Using image registry.k8s.io/ingress-nginx/kube-webhook-certgen:v20230312-helm-chart-4.5.2-28-g66a760794
🔎  Verifying ingress addon...
🌟  The 'ingress' addon is enabled
$ kubectl -n ingress-nginx get configmaps tcp-services -o yaml
apiVersion: v1
kind: ConfigMap
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"v1","kind":"ConfigMap","metadata":{"annotations":{},"labels":{"app.kubernetes.io/component":"controller","app.kubernetes.io/instance":"ingress-nginx","app.kubernetes.io/name":"ingress-nginx"},"name":"tcp-services","namespace":"ingress-nginx"}}
  creationTimestamp: "2023-05-25T13:26:33Z"
  labels:
    app.kubernetes.io/component: controller
    app.kubernetes.io/instance: ingress-nginx
    app.kubernetes.io/name: ingress-nginx
  name: tcp-services
  namespace: ingress-nginx
  resourceVersion: "462"
  uid: 3ac05bda-1ae2-439c-90c5-8d47dcdfb5dd
$ kubectl apply -f  docs/src/samples/cluster-expose-service.yaml
configmap/tcp-services configured
service/ingress-nginx created
$ kubectl -n ingress-nginx get configmaps tcp-services -o yaml
apiVersion: v1
data:
  "5432": default/cluster-example-rw:5432
kind: ConfigMap
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"v1","data":{"5432":"default/cluster-example-rw:5432"},"kind":"ConfigMap","metadata":{"annotations":{},"name":"tcp-services","namespace":"ingress-nginx"}}
  creationTimestamp: "2023-05-25T13:26:33Z"
  name: tcp-services
  namespace: ingress-nginx
  resourceVersion: "1598"
  uid: 3ac05bda-1ae2-439c-90c5-8d47dcdfb5dd
$ kubectl patch configmap tcp-services -n ingress-nginx --patch '{"data":{"5432":"default/cluster-example-rw:5432"}}'
configmap/tcp-services patched (no change)
```
As you can see, the `kubectl patch` it's not changing anything since it's the same that was patched by the `cluster-expose-service.yaml` file

Reply:
Completely get it - I was going on the principle that the section needed to exist, as is. But as you point out, mini line is now sufficiently in sync as to make having a separate section like this possibly confusing to the reader. 

The Testing on minikube was effectively a separate walk through with a different patch file being used to expose just one port. Removing the top-services patch would break that but at the same time the testing pathway doesn’t direct the reader to use the yank file described earlier. 

It could be enough to just mention the Addison’s enable for minikube when mentioning enabling ingress and leave the rest untouched, losing the testing section?

as it stands though, the current fix offered corrects the errors in the current version without disturbing the existing content structure. I leave it to your judgement as to whether this is worth merging as is or whether a more extensive reworked PR is necessary.

Reply:
I added the suggested change using the GitHub interface